src/midi/fluid_midi.c

```diff
@@ -2080,18 +2089,30 @@ fluid_player_callback(void *data, unsigned int msec)
     int i;
     int loadnextfile;
     int status = FLUID_PLAYER_DONE;
+    fluid_midi_event_t mute_event;
     fluid_player_t *player;
     fluid_synth_t *synth;
     player = (fluid_player_t *) data;
     synth = player->synth;
 
     loadnextfile = player->currentfile == NULL ? 1 : 0;
+    
+    fluid_midi_event_set_type(&mute_event, CONTROL_CHANGE);
+    mute_event.param1 = ALL_SOUND_OFF;
+    mute_event.param2 = 1;
 
     if(fluid_player_get_status(player) != FLUID_PLAYER_PLAYING)
     {
         if(fluid_atomic_int_get(&player->stopping))
         {
-            fluid_synth_all_notes_off(synth, -1);
+			for(i = 0; i < synth->midi_channels; i++)
```

Comment:
Some more tabs here to remove :-) (and further down)

Reply:
They keep sneaking in! thanks